release/models/rib/openconfig-rib-bgp-attributes.yang

```diff
@@ -81,6 +87,12 @@ submodule openconfig-rib-bgp-attributes {
     description
       "Data for representing BGP AS-PATH attribute";
 
+    leaf index {
+      type uint64;
```

Comment:
uint32 is plenty here.  Very likely uint16 is sufficient for standard RFC 4271 4k max pdu packets, but with extended 65k packets with extension uint16 may not be sufficient for pedantic implementation.  In practice, segment list size will be small.

Reply:
Does this comment still hold given the other reply?

Reply:
Changed to uint32